From 3557a79f14e46b74f2992fa4b6aaedda82c13c6f Mon Sep 17 00:00:00 2001
From: Martin Klapetek <mklapetek@kde.org>
Date: Thu, 16 Jul 2015 14:52:21 +0200
Subject: [PATCH 2/8] Remove pretty much useless warning message

This gets printed on every notification close, even unrelated
notifications not created/handled by the current plugin, creating
unnecessary spam in everything having a KNotification.
---
 src/notifybypopup.cpp | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/notifybypopup.cpp b/src/notifybypopup.cpp
index e377051..bb96465 100644
--- a/src/notifybypopup.cpp
+++ b/src/notifybypopup.cpp
@@ -465,7 +465,6 @@ void NotifyByPopup::onGalagoNotificationActionInvoked(uint notificationId, const
 {
     auto iter = d->galagoNotifications.find(notificationId);
     if (iter == d->galagoNotifications.end()) {
-        qWarning() << "Failed to find KNotification id for dbus_id" << notificationId << "- action not triggered";
         return;
     }
 
@@ -484,7 +483,6 @@ void NotifyByPopup::onGalagoNotificationClosed(uint dbus_id, uint reason)
 {
     auto iter = d->galagoNotifications.find(dbus_id);
     if (iter == d->galagoNotifications.end()) {
-        qWarning() << "Failed to find KNotification for dbus_id" << dbus_id;
         return;
     }
     KNotification *n = *iter;
-- 
1.9.3

